scripts/create-clusters.sh

```diff
@@ -155,7 +164,7 @@ function check-clusters-ready() {
 }
 
 function configure-insecure-registry-on-cluster() {
-  configure-insecure-registry-and-reload "docker exec cluster${1}-control-plane bash -c" '$(pgrep dockerd)'
+  configure-insecure-registry-and-reload-for-containerd "docker exec cluster${1}-control-plane bash -c" '$(pgrep -x containerd)'
```

Comment:
Given that `configure-insecure-registry-and-reload-for-containerd` is only used for `configure-insecure-registry-on-cluster`, is there a reason not to inline it here?